ietf-wdm-interface.yang

```diff
@@ -181,78 +98,62 @@ module ietf-ext-xponder-wdm-if {
         description "Q Factor TCA";
       }
     }
-    description "The different types of TCA's";
+    description
+      "The different types of TCA's for DWDM Interfaces";
   }
 
-  grouping opt-if-och-power {
-    description "Interface optical Power";
-
-    leaf channel-output-power {
-      type l0-types:power-dbm-or-null;
-      description "The output power for this interface in dBm.
-      The setting of the output power is optional";
-    }
-
-    leaf channel-input-power {
-      type l0-types:power-dbm-or-null;
-      config false;
-      description "The current channel input power of this
-      interface";
-    }
-
-    leaf total-input-power {
-      type l0-types:power-dbm-or-null;
-      config false;
-      description "The total input power of this interface";
-    }
-  }
 
-  grouping opt-if-och-tca-thresholds {
+  grouping wdm-if-tca-thresholds {
     description "Thresholds for TCA's";
     leaf tca-type {
-      type opt-if-och-tca-types;
+      type wdm-if-tca-types;
       description "type of the TCA eg TX Power";
     }
     leaf min-threshold {
       type int32;
-      description "A TCA is generated if the variable is less than
-      this value";
+      description
+        "A TCA is generated if the variable is less than
+        this value";
     }
     leaf max-threshold {
       type int32;
-      description "A TCA is generated if the variable is more than
-      this value";
+      description
+        "A TCA is generated if the variable is more than
+        this value";
     }
     leaf threshold-hysteresis {
       type int32;
-      description "The TCA raise/clear hysteresisis around the
-      threshold value to avoi alert flikering";
+      description
+        "The TCA raise/clear hysteresisis around the
+        threshold value to avoi alert flikering";
     }
   }
 
-  grouping opt-if-och-central-frequency {
+  grouping wdm-if-central-frequency {
```

Comment:
I am wondering whether we can have a common grouping to define the carrier central frequency and use it in optical impairments, WDM tunnel and this models